### ISSUE TITLE
~/.bashrc doesn't exist by default?

### ISSUE THREAD

**Issue by andrewhoff** [bold-commerce/dev-workstation#4]
issues encountered when trying to source ~/.bash_profile after adding GOPATH stuff

**Comment by markstgodard** [comment]
Good question, was this from a fresh MacOS install? We could update `setup` to possibly 
touch .bash_profile if it doesn't exist

We've found cases were running setup from scratch on newer versions of MacOS gave us difference results ;) So we can investigate

**Comment by andrewhoff** [comment]
Yes, sorry. Was in a hurry. ~/.bashrc on macOS didn't seem to exist on the fresh install on this individual's laptop
